python/tank/bootstrap/manager.py

```diff
@@ -516,7 +516,21 @@ def prepare_engine(self, engine_name, entity):
 
         return path
 
-    def get_pipeline_configurations(self, project):
+    def get_resolved_pipeline_configuration_descriptor(self, project_id=None):
```

Comment:
this name is very long. Could we come up with something shorter? `resolve_descriptor()`?